simulations/vip-460/zksyncmainnet.ts

```diff
@@ -0,0 +1,145 @@
+      });
+
+      it(`should register ${newMarket.vToken.symbol} in PoolRegistry`, async () => {
+        const registeredVToken = await poolRegistry.getVTokenForAsset(COMPTROLLER, newMarket.vToken.underlying.address);
+        expect(registeredVToken).to.equal(newMarket.vToken.address);
+      });
+    });
+
+    describe("Risk parameters", () => {
+      checkRiskParameters(newMarket.vToken.address, newMarket.vToken, newMarket.riskParameters);
+
+      it("should pause brrowing on zkETH", async () => {
+        expect(await comptroller.actionPaused(newMarket.vToken.address, 2)).to.equal(true);
+      });
+    });
+
+    describe("Ownership and initial supply", () => {
+      const { vToken: vTokenSpec, initialSupply } = newMarket;
+      const vTokenContract = new ethers.Contract(vTokenSpec.address, VTOKEN_ABI, provider);
+
+      describe(`${vTokenSpec.symbol}`, () => {
+        it(`should have owner = normal timelock`, async () => {
+          expect(await vTokenContract.owner()).to.equal(NORMAL_TIMELOCK);
+        });
+
+        const vTokenSupply = convertAmountToVTokens(initialSupply.amount, vTokenSpec.exchangeRate);
+        const vTokenSupplyForReceiver = vTokenSupply.sub(initialSupply.vTokensToBurn);
+        const format = (amount: BigNumber, spec: { decimals: number; symbol: string }) =>
+          `${formatUnits(amount, spec.decimals)} ${spec.symbol}`;
+
+        it(`should have balance of underlying = ${format(initialSupply.amount, vTokenSpec.underlying)}`, async () => {
+          const underlying = new ethers.Contract(vTokenSpec.underlying.address, ERC20_ABI, provider);
+          expect(await underlying.balanceOf(vTokenSpec.address)).to.equal(initialSupply.amount);
+        });
+
+        it(`should have total supply of ${format(vTokenSupply, vTokenSpec)}`, async () => {
+          expect(await vTokenContract.totalSupply()).to.equal(vTokenSupply);
+        });
+
+        it(`should send ${format(vTokenSupplyForReceiver, vTokenSpec)} to receiver`, async () => {
+          const receiverBalance = await vTokenContract.balanceOf(initialSupply.vTokenReceiver);
+          expect(receiverBalance).to.equal(vTokenSupplyForReceiver);
+        });
+
+        it(`should burn ${format(initialSupply.vTokensToBurn, vTokenSpec)}`, async () => {
+          const burnt = await vTokenContract.balanceOf(ethers.constants.AddressZero);
+          expect(burnt).to.equal(initialSupply.vTokensToBurn);
+        });
+
+        it(`should leave no ${vTokenSpec.symbol} in the timelock`, async () => {
+          const timelockBalance = await vTokenContract.balanceOf(NORMAL_TIMELOCK);
+          expect(timelockBalance).to.equal(0);
+        });
+      });
+    });
+
+    describe("Interest rates", () => {
+      checkInterestRate(
+        newMarket.interestRateModel.address,
+        newMarket.vToken.symbol,
+        newMarket.interestRateModel,
+        BLOCKS_PER_YEAR,
+      );
+    });
+
+    checkIsolatedPoolsComptrollers();
```

Comment:
Tests are failing for me, I am getting error `Error function_selector = 0xdb33be3d`